sample-spring-boot-echo/src/main/resources/application-template.yml

```diff
@@ -1,28 +1,12 @@
-#
-# Copyright 2016 LINE Corporation
-#
-# LINE Corporation licenses this file to you under the Apache License,
-# version 2.0 (the "License"); you may not use this file except in compliance
-# with the License. You may obtain a copy of the License at:
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-# License for the specific language governing permissions and limitations
-# under the License.
-
-This is tempalte file for configuration.
-Please copy as `application.yml` and change your value.
+server.use-forward-headers: true
+server:
+  port: 7777
 
 line.bot:
-  channel-token: 'Put Your Channel Token Here.'
-  channel-secret: 'Put Your Channel Secret Here.'
   handler.path: /callback
+  channel-token: ${your-channel-token-1}
+  channel-secret: ${your-channel-secret-1}
+  list :
+    - channel-token: ${your-channel-token-2}
```

Comment:
I used the above configuration to make it compatible with existing APIs.
I also think it is ideal to manage all under the list.